openwisp_monitoring/device/tasks.py

```diff
@@ -5,17 +5,20 @@
 
 
 @shared_task
-def trigger_device_checks(pk):
+def trigger_device_checks(pk, recovery=True):
```

Comment:
This might be better as we can choose as to what should happen in case checks don't exist. Maybe the parameter `recovery` could be renamed as it seems slightly misleading to me but I can't think of a better name right now.

Reply:
if another part of openwisp detects a recovery (eg: management VPN status is up again) it will pass True, otherwise it will pass False, it's also consistent with "device recovery detection".